scalalib/src/mill/scalalib/PublishModule.scala

```diff
@@ -275,7 +283,7 @@ object PublishModule extends ExternalModule {
       publishArtifacts: mill.main.Tasks[PublishModule.PublishData],
       sonatypeCreds: String = "",
       signed: Boolean = true,
-      gpgArgs: String = defaultGpgArgs.mkString(","),
+      gpgArgs: String = defaultStringGpgArgs,
```

Comment:
I tried to standardize anything that was being used in multiple times and this is being used in three different places. I've made defaultStringGpgArgs `private [mill]`, so any refactoring won't break bincompat later on.

Reply:
The issue is, that users who want to override this methods might want to re-use the default arguments. Since these are now pre-formatted and private, they no longer can't. Whereas previously, they could just re-use the `Seq`, maybe filter some args and add others and are done. Theoretically, they still could use `defaultGpgArgs`, but there is nowhere any indication of that option, not in the docs nor in the visible (to them) code.

Hence, I think keeping `defaultGpgArgs` is the better choice. Even if that means, we need to format them (`.mkString(","`) in multiple places. But the API is more clear and obvious defaults are reusable. And this is not a time-critical code path.

```suggestion
      gpgArgs: String = defaultGpgArgs.mkString(","),
```

Reply:
Done.